aptos-move/framework/aptos-names/sources/config.move

```diff
@@ -288,6 +325,13 @@ module aptos_names::config {
         property_map::read_bool(&borrow_global<ConfigurationV1>(addr).config, key)
     }
 
+    public fun read_u8_vector_v1(addr: address, key: &String): vector<u8> acquires ConfigurationV1 {
+        let value = property_map::borrow_value(property_map::borrow(&borrow_global<ConfigurationV1>(addr).config, key));
+        // remove the length of this vector recorded at index 0
+        vector::remove(&mut value, 0);
```

Comment:
where is this length coming from? This seems to introduce unexpected behaviour if you ever want to store a vecu8 that does NOT have length at 0, no?

Reply:
like alin mentioned, the property map doesn't return the vector itself, but instead the bcs serialization of the vector, which has its length at index 0.